packages/kbn-optimizer/limits.yml

```diff
@@ -108,4 +108,4 @@ pageLoadAssetSize:
   fileUpload: 25664
   banners: 17946
   mapsEms: 26072
-  cases: 102558
+  cases: 162385
```

Comment:
We're really trying to get each plugin under 100k, and ideally much less, as this size represented here is loaded on every request. We have documentation on how to achieve this with lazy loading https://www.elastic.co/guide/en/kibana/master/plugin-performance.html

Reply:
Thanks @tylersmalley! We're actually working on porting code out of `security_solution` so a lot of this is from that. Once all the code is ported, we'll be removing it from security solution and the overall bundle size should remain relatively unchanged.

Reply:
If that is the case, can we remove the ~60k from the securitySolutions plugin as part of this change? And I would really ask that implementing async chunks would be prioritized, as this impacts all of Kibana.

Reply:
Yea, I wouldn't mind doing that, it was just our original plan to move everything out of security solution once this migration is done rather that piecemeal. This work isn't being backported to `7.x` until everything is ported tested and removed from `security solution`. We are currently using `React.lazy` to export these components as cases isn't going to be a registered application atm: https://github.com/michaelolo24/kibana/blob/move_all_cases_page/x-pack/plugins/cases/public/plugin.ts 


@XavierM thoughts?

Reply:
I am not suggesting that the async work needs to be blocked here, just that the limit added to cases be removed from securitySolution.

I am curious though, why not backport pull requests as they are merged? Not doing that seems like it would create a lot of issues down the road.

Reply:
hey, we were trying something new (mean I had this great idea) but after talking with few people and between us. We are ok to NOT rock the boat or get our boat sinking. And we will go the route of a feature branch to be safer.

Reply:
@michaelolo24 here our feature branch https://github.com/elastic/kibana/tree/cases_rac_ui